crates/dht/src/arc_set.rs

```diff
@@ -114,6 +114,22 @@ impl ArcSet {
         self.inner.len()
     }
 
+    /// Create an [ArcSet] from a list of sectors indices.
+    ///
+    /// This is intended to be used for constructing an [ArcSet] from a network message.
+    pub fn from_raw(inner: impl Iterator<Item = u32>) -> Self {
```

Comment:
Same deal here with sending over the network, I want to be able to reconstruct these and am happy to be pointed at a better method.

One option would just be to serialize the type and that avoids exposing the implementation, for a little extra brain power required to understand all the levels of nested serialization

Reply:
If we're pretty sure we're going to stick with the 512 indefinitely, what do you think about packing these into the individual bits of 64 bytes rather than the potentially 4096 bytes of 512 u32s?

Reply:
Done -> https://github.com/holochain/kitsune2/pull/72/commits/8a54415ed535282a99c6f4381b7dce2e19e768dd